added titles to the docs pages (#109)
dappforce/subsocial-docs: docs/develop/sdk/reactions/CreateReactions.md

title: Create, Update, And Delete Reactions
displayed_sidebar: developSidebar
---

<head>

>   <title>Create Reactions on Subsocial | JS SDK Guide</title>

Reviewer: ```suggestion
  <title>How To Create Reactions With The Subsocial JS SDK</title>
```
